### ISSUE TITLE
bug in shallow populate function when object includes "data" field  

### ISSUE THREAD

**Issue by aliciahogue** [marshallswain/feathers-graph-populate#19]
my user is set up with with a json column named data, this line below is what is causing all my confusion I believe!   ( things working in some cases and not in others, leading me to assume its my own code issues lol )  

In my case, it should be taking context.result to access the user object, as context.result.data is a field of my user so the dataMap function is never finding the keys to match up.

```js
return function shallowPopulate (context) {
    const { app, type } = context
    let data = type === 'before' ? context.data : (context.result.data || context.result)
    data = [].concat(data || [])

    if (!data.length) {
      return Promise.resolve(context)
    }


```

**Comment by marshallswain** [comment]
@aliciahogue I think this is the line of code: https://github.com/Mattchewone/feathers-shallow-populate/blob/master/lib/shallow-populate.js#L54

Could you try opening the file in your `node_modules` folder and replacing that line with the following snippet:

```js
let data = type === 'before' 
    ? context.data : 
    context.method === 'get'
    ? context.result :
    (context.result.data || context.result)
```

**Comment by aliciahogue** [comment]
Done!! First time I have ever done a pull request hahaha.  Yay me.  Pretty sure this library is going to be my new favorite addition to my project!!

**Comment by marshallswain** [comment]
I realize that I made a short sighted mistake in that logic.  We should change it to this

```

let data = type === 'before' 
    ? context.data : 
    context.method === 'find'
    ? (context.result.data || context.result)
    : context.result
   

```

**Comment by aliciahogue** [comment]
This was done awhile back, I had commented on the pull request asking if there was anything else needed for it to be released?  Thinking the comments may have gotten missed so I am posting again here haha.

**Comment by marshallswain** [comment]
@aliciahogue this is now fixed in the just-published `feathers-graph-populate@1.2.0`.  Sorry that took so long!  Thank you for the reminder.
